build/yamls/kube-proxy-windows.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
```

Comment:
does this file differ from https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml, and if yes how?

Reply:
The difference is the params for kube-proxy.exe. For antrea, we use `userspace mode` of kube-proxy.  The version of kubernetes-sigs use the `kernel mode`.

Reply:
thanks for the clarification

Reply:
A different question - this file is used for CI/testing only? Or we assume user should use this yaml to deploy kube-proxy? If not needed, I would not maintain the kube-proxy yaml in our repo, and track all changes in upstream.

Should we put it in a separate dir, if it is not part of antrea deployment?

Reply:
Hi @jianjuns, The `kube-proxy-windows.yaml` is to be provided to users. The reason why we maintain it is because upstream just provides a kube-proxy yaml for `kernel mode`, But we `userspace mode` for kube-proxy.

Maybe we can considered to submit the file as PR on upstream to support kube-proxy userspace mode.

Reply:
Ok. Should we consider to download and patch the upstream yaml file?
And we should document user can also change the upstream yaml themselves, instead of using our yaml.

Reply:
> Ok. Should we consider to download and patch the upstream yaml file?
> And we should document user can also change the upstream yaml themselves, instead of using our yaml.

Yes Jianjun. I think it make sence. I will remove `kube-proxy-windows.yaml` and add changes needed for user in doc.